addons/website_sale/static/src/scss/product_configurator.scss

```diff
@@ -1,31 +1,26 @@
 .css_attribute_color {
+    --css-attribute-color-border-width: .25rem;
+
     position: relative;
     display: inline-block;
-    border: 5px solid $input-border-color;
+    border: var(--css-attribute-color-border-width) solid $body-bg;
+    width: calc(2rem - var(--css-attribute-color-border-width));
+    height: calc(2rem - var(--css-attribute-color-border-width));
     border-radius: 50%;
-    text-align: center;
     transition: $input-transition;
-
-    @include o-field-pointer();
-
-    &:before {
-        content: "";
-        display: block;
-        @include o-position-absolute(-3px, -3px, -3px, -3px);
-        border: 4px solid white;
-        border-radius: 50%;
-        box-shadow: inset 0 0 3px rgba(black, 0.3);
-    }
+    outline: $border-width solid $input-border-color;
+    box-shadow: inset 0 0 0 1px rgba($dark, .1);
+    box-sizing: content-box;
 
     input {
-        margin: 8px;
-        height: 13px;
-        width: 13px;
         opacity: 0;
     }
 
-    &.active {
-        border: 5px solid map-get($theme-colors, 'primary');
+    &.active, &:hover {
```

Comment:
These buttons are lacking a focus state. We should ensure that are navigable using the keyboard